### PR TITLE
[utils] Fix bug of imagenet-process.py

### PR DESCRIPTION
**Pull request by whiteking64**
Summary:
+ Fix https://github.com/pytorch/glow/issues/5764
+ Make path operations safer & better

### CODE REVIEW

File: utils/imagenet-process.py [hunk 1]
```diff
@@ -37,34 +37,36 @@
 
 # create the output dir if necessary
 try:
-    os.mkdir(args.output)
-except Exception:
-    pass
+    os.makedirs(args.output, exist_ok=True)
+except Exception as e:
+    raise print(e)
```
Comment (khabinov) [inline utils/imagenet-process.py:42]: Sorry if I'm missing something here, but what does `raise print(e)` do?

File: utils/imagenet-process.py [hunk 2]
```diff
@@ -37,34 +37,36 @@
...
-    im.convert("RGB")
+    im = im.convert("RGB")
     resize = torchvision.transforms.Compose(
         [torchvision.transforms.Resize(256), torchvision.transforms.CenterCrop(224)]
     )
     processed_im = resize(im)
 
     if args.normalize:
-        normalize = torchvision.transforms.Compose(
+        trans = torchvision.transforms.Compose(
```
Comment (khabinov) [inline utils/imagenet-process.py:58]: Nit: maybe better to call it `transform_fn` or something similar.

**Review by khabinov — COMMENTED** [review]
I think the change looks good.

File: utils/imagenet-process.py (hunk 2 above)
Comment (whiteking64) [reply utils/imagenet-process.py]: Thank you for the suggestion!
d40880966c771bf1ed5ff3962c72dc6245d64987

**Review by khabinov — COMMENTED** [review]
LGTM.

**Review by houseroad — APPROVED** [review]

**Pull request closed without merging**